src/js/engagement_view/src/components/reusableComponents/graplHeader/GraplHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import Home from '@material-ui/icons/Home';
+
+import {graplHeaderStyles} from './styles';
+import {GraplHeaderProps} from 'types/GraplHeaderTypes'; 
+
+const useStyles = graplHeaderStyles; 
+
+const GraplHeader = (
+    {displayBtn}: GraplHeaderProps) => {
```

Comment:
sort of odd way to do function signature (with the leading parens on a diff line)

Reply:
Agreed and resolved.